README.md

```diff
@@ -29,6 +32,21 @@ cd omero-guide-python
 repo2docker .
 ```
 
+Building locally using conda and Jupyter:
+
+ * Create a conda environment i.e. ``conda env update -n stardist -f binder/environment.yml``
+ * Activate the newly created environment
+
+The following steps are only required if you want to run the notebooks
+ * If you have Anaconda installed:
+   * Start Jupyter from the Anaconda-navigator
+   * Select the notebook you wish to run and select the ``Kernel>Change kernel>Python [conda env:stardist]``
+ * If Anaconda is not installed:
+   * In the environment, install ``jupyter`` e.g. ``pip install jupyter``
+   * Add the virtualenv as a jupyter kernel i.e. ``ipython kernel install --name "stardist" --user``
+   * Open jupyter notebook i.e. ``jupyter notebook`` and select the ``stardist`` kernel if set or ``[conda env:stardist]`` if available
```

Comment:
```suggestion
   * Open jupyter notebook i.e. ``jupyter notebook`` and select the ``stardist`` kernel or ``[conda env:stardist]`` according to what is available
```

Reply:
suggesting that above as was not sure why to use two wordings (``if set`` and ``if available``).
Reject if not appropriate